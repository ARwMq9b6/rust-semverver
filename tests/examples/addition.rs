pub mod old {
    #[allow(dead_code)]
    struct Bcd;

    mod b {

    }

    mod c {

    }
}

pub mod new {
    pub struct Abc;

    pub struct Bcd;

    pub mod a {

    }

    pub mod b {
        #[allow(dead_code)]
        pub struct Cde;
    }

    mod c {
        #[allow(dead_code)]
        pub struct Def;
    }
}
