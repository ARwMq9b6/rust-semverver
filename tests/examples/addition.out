version bump: 1.0.0 -> (TechnicallyBreaking) -> 1.1.0
warning: Addition of at least one path to `Abc` (TechnicallyBreaking)
  --> $REPO_PATH/tests/examples/addition.rs:15:5
   |
15 |     pub struct Abc;
   |     ^^^^^^^^^^^^^^^

warning: TechnicallyBreaking changes in `Bcd`
  --> $REPO_PATH/tests/examples/addition.rs:17:5
   |
17 |     pub struct Bcd;
   |     ^^^^^^^^^^^^^^^
   |
   = note: TechnicallyBreaking (ItemMadePublic)

warning: Addition of at least one path to `a` (TechnicallyBreaking)
  --> $REPO_PATH/tests/examples/addition.rs:21:5
   |
21 |     }
   |     ^

warning: TechnicallyBreaking changes in `b`
  --> $REPO_PATH/tests/examples/addition.rs:24:9
   |
24 | /         #[allow(dead_code)]
25 | |         pub struct Cde;
26 | |     }
   | |_____^
   |
   = note: TechnicallyBreaking (ItemMadePublic)

warning: Addition of at least one path to `Cde` (TechnicallyBreaking)
  --> $REPO_PATH/tests/examples/addition.rs:25:9
   |
25 |         pub struct Cde;
   |         ^^^^^^^^^^^^^^^

